Draggable/explainer.md

```diff
@@ -0,0 +1,165 @@
+* [Confusion on how the feature works](https://github.com/electron/electron/issues/1354).
+
+## Property Values
+
+The `app-region` property has 2 possible values:
+* `drag`: marks an element as a draggable area. Clicking and dragging on the element moves the entire window on the screen.
+* `no-drag`: cancels the draggable behaviour in a nested element.
+
+## Current property behaviour
+
+The `app-region` property works on `inline` and `block` elements. There is one requirement for the feature to work on web apps and that is to have the application running in a standalone window and with a `display-override` value of `['window-controls-overlay']`. The reason for this is that when in this override, the host window loses its titlebar, which is the only area that allows an end user to move the window in the screen.
+
+Below is an illustration of such case, showing the [WCO](https://developer.mozilla.org/en-US/docs/Web/API/Window_Controls_Overlay_API) feature toggled _on_:
+
+![Web App with WCO enabled and an animated button](image1.png)
+
+And here is the HTML code for the example:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>test</title>
+    <link rel="manifest" href="manifest.json">
+    <style>
+        button {
+            font-size: 1.5rem;
+            animation-name: moveDown;
+            animation-duration: 120s;
+            animation-iteration-count: infinite;
+            app-region: drag; /* <-------- APP-REGION */
+        }
+
+        @keyframes moveDown {
+            from { margin-top: 0px; }
+            to { margin-top: 600px;}
+        }
+    </style>
+</head>
+<body>
+    <button title="should be draggable">
+        Generic Beautiful Button
+    </button>
+    
+</body>
+</html>
+```
+
+In this example, the 'WCO' feature is enabled, hence the window has lost the meaningful area to be moved across the screen.
+
+The (Generic Beautiful) button has the `app-region` set to `drag`. This button can how be used to drag the window around.
+
+> Note: This is just an example to test the _current_ behaviour on elements that have animations. the `app-region` is generally used on emulated titlebars or other UX designed to let the end user move the window around.
+
+### `app-region` and pointer events
+
+Current implementations of `app-region` **ignore pointer events**. Interactive elements inside an area defined with `app-region: drag` will require a `app-region: no-drag` to be interacted with.
+
+### Suppression of `app-region`
+
+The `app-region` property applies to a DOM element, and allows it to become a point from where to drag and move a window. If this element's `visibility` is `hidden` or `collapsed`, or if the element is occluded by an element with a higher `z-index` then the window can't be dragged from that element.
+
+### `app-region` and animations
```

Comment:
Can we either broaden this section to include all modifications/suppressions (if they all fit nicely together), or add new sections to cover them?

I'm thinking about cases like the `app-region` property being dynamically changed mid-drag, or any of the things called out in the section above this.

Reply:
I will keep adding more use cases as they appear throughout the discussion of this feature.